package_build_list_host_darwin.json

```
@@ -12,8 +12,8 @@
         "AwsIotDeviceSdkCpp-1.15.2-rev2-mac": "Scripts/extras/pull_and_build_from_git.py ../../package-system/AwsIotDeviceSdkCpp --platform-name Mac --package-root ../../package-system --clean",
         "freetype-2.11.1-rev1-mac": "Scripts/extras/pull_and_build_from_git.py ../../package-system/freetype --platform-name Mac --package-root ../../package-system/freetype/temp --clean",
         "freetype-2.11.1-rev1-ios": "Scripts/extras/pull_and_build_from_git.py ../../package-system/freetype --platform-name iOS --package-root ../../package-system/freetype/temp --clean",
-        "googlebenchmark-1.5.0-rev2-ios": "Scripts/extras/pull_and_build_from_git.py ../../package-system/googlebenchmark --platform-name iOS --package-root ../../package-system --clean",
-        "googlebenchmark-1.5.0-rev2-mac": "Scripts/extras/pull_and_build_from_git.py ../../package-system/googlebenchmark --platform-name Mac --package-root ../../package-system --clean",
+        "googlebenchmark-1.7.0-rev1-ios": "Scripts/extras/pull_and_build_from_git.py ../../package-system/googlebenchmark --platform-name iOS --package-root ../../package-system --clean",
+        "googlebenchmark-1.7.0-rev1-mac": "Scripts/extras/pull_and_build_from_git.py ../../package-system/googlebenchmark --platform-name Mac --package-root ../../package-system --clean",
```

--package-root is not needed if the 'build_from_folder' value is updated accordingly (see below)
```suggestion
        "googlebenchmark-1.7.0-rev1-ios": "Scripts/extras/pull_and_build_from_git.py ../../package-system/googlebenchmark --platform-name iOS  --clean",
        "googlebenchmark-1.7.0-rev1-mac": "Scripts/extras/pull_and_build_from_git.py ../../package-system/googlebenchmark --platform-name Mac  --clean",
```